src/components/media-player/volume-presets-modal.js

```diff
@@ -89,12 +90,21 @@ function VolumePresetsModal({ volumePresets, isOpen, onOk, onClose }) {
     );
   };
 
+  const isModalOkButtonDisable = managedVolumePresets.some(preset => !preset.name.trim());
+
   return (
     <Modal
       title={t('modalTitle')}
       open={isOpen}
-      onOk={handleOk}
       onCancel={handleCancel}
+      footer={[
+        <Button key="cancel" onClick={handleCancel}>
+          {t('common:cancel')}
+        </Button>,
+        <Button key="ok" type="primary" disabled={isModalOkButtonDisable} onClick={handleOk}>
+          {t('common:ok')}
+        </Button>
+      ]}
       >
       <div className="u-modal-body">
         {managedVolumePresets.map(renderVolumePreset)}
```

Comment:
Tiny inconsistency: could we disable the icon rendered by `renderVolumePreset` when there's only one preset left (and thus this last one cannot be deleted)?

Reply:
Done, good that you spotted this, it was a forgotten case in the `delete-button` component.

Reply:
:+1: